modules/electrophysiology_browser/jsx/components/electrophysiology_session_panels.js

```diff
@@ -251,6 +252,23 @@ class FilePanel extends Component {
                   </a>
                 </div>
               </div>
+              <div className={'form-group row flex-v-center'}>
+                <div className={'col-xs-5'}
+                     style={stylesFile.div.element.download_title}
+                >Annotation Files</div>
```

Comment:
Move this below "Events" button and change download behaviour to match Events (single file?) 
```suggestion
                <div>Annotations</div>
```